SoftLayer/managers/vs.py

```diff
@@ -1002,6 +1003,25 @@ def _get_price_id_for_upgrade_option(self, upgrade_prices, option, value, public
                 else:
                     return price.get('id')
 
+    def get_summary_data_usage(self, instance_id, start_date=None, end_date=None, valid_type=None, summary_period=None):
```

Comment:
This command should be finding the Metric_tracking_object id from the VS id. 

```
./slcli vs usage -s 2019-03-01 -e 2019-04-01 -t CPU0 -summary 600 73403809
SoftLayerAPIError(404): Unable to find object with id of '73403809'.
~/Code/softlayer-python  $ ./slcli vs detail 73403809
:....................:......................................:
:               name : value                                :
:....................:......................................:
:                 id : 73403809                             :
:               guid : ee8813d2-8c30-41a2-b915-026b7f5da226 :
:           hostname : test                                 :
:....................:......................................:
```

Reply:
Done.